root/static/scripts/common/utility/storage.js

```diff
@@ -19,7 +19,7 @@ try {
 }
 
 // Holds data where localStorage isn't supported
-const store: {[string]: string, ...} = {};
+const store: {[name: string]: string, ...} = {};
```

Comment:
`name` can be found here https://github.com/metabrainz/musicbrainz-server/blob/master/root/static/scripts/common/utility/storage.js#L62